telemetry/src/Transmission/Transmission.cpp

```diff
@@ -152,10 +159,12 @@ void Transmission::enableTransmission() {
 
 void Transmission::disableTransmission() {
   /* Wait until done transmitting / receiving*/
-  while (busyTransmitting)
-    ;
+  while (busyTransmitting) {
+  };
```

Comment:
Isnt this dangerous?

Reply:
The behavior is the same both with and without brackets, so it's as dangerous as it was :)